.github/workflows/ci.yml

```diff
@@ -43,36 +65,57 @@ jobs:
           invoke type-check --ci --pretty
           invoke type-check --ci --pretty  --check-stub-sources
 
-  docs:
-    if: github.event.pull_request.draft == false
+  docs-snippets:
+    name: docs-snippets
+    #if: github.event.pull_request.draft == false
+    #add more scope here
     runs-on: ubuntu-latest
-
+    env:
+      GE_TEST_GCP_CREDENTIALS: ${{secrets.GE_TEST_GCP_CREDENTIALS}}
+      GE_TEST_GCP_PROJECT: ${{secrets.GE_TEST_GCP_PROJECT}}
+      GE_TEST_BIGQUERY_DATASET: ${{secrets.GE_TEST_BIGQUERY_DATASET}}
+      GOOGLE_APPLICATION_CREDENTIALS: 'gcp-credentials.json'
```

Comment:
A bit hacky, but needed for authorizing access to Google Cloud with a service account. Otherwise the credentials just lives as a JSON file in a ENV variable